app/tests/admin/governance/dids/test_dids.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from admin.governance.dids import get_trusted_registry, get_trusted_partner
+
+APPLICATION_JSON_CONTENT_TYPE = {"content-type": "application/json"}
+BASE_PATH = "/admin/governance/dids"
+
+
+@pytest.mark.asyncio
+async def test_get_trusted_registry(async_client, yoma_agent):
+
+    response = await async_client.get(
+        BASE_PATH + "/trusted-registry",
+        headers={"x-api-key": "adminApiKey", **APPLICATION_JSON_CONTENT_TYPE},
+    )
+    assert response.status_code == 200
+    result = response.json()
+    expected_keys = ["did", "posture", "verkey"]
+    assert [list(res.keys()) == expected_keys for res in result]
+    assert len(result) == 1
+
+    res_method = await get_trusted_registry(aries_controller=yoma_agent)
+    assert res_method == result
+
+
+@pytest.mark.asyncio
+async def test_get_trusted_partner(async_client, yoma_agent):
+
+    partner_id = "jhoLFYW6LfMEPUEZWR18A"  # YOMA
```

Comment:
Tests need to run as independently as possible and thus where possible not not depend on some data already being present. Tests need to be built as robust as possible.

We also cannot depend on the order of tests being predictable.

There is at least [existing code for adding a public did in the test context](https://github.com/didx-xyz/aries-cloudapi-python/blob/d2280a173bbe383c9b5988b59b5bbff607bc4d7a/app/tests/admin/governance/schemas/test_schemas.py#L96). Possible be worth extracting that into a fixture.

Reply:
agreed!

Reply:
any suggestions for this specific case?

Reply:
@michaelwiles